### ISSUE TITLE
Create project with underscore issue?

### ISSUE THREAD

**Issue by rnaszcyn** [openshiftio/launchpad-frontend#61]
If I create a new project based on Swarm HTTP API and use a project name like swarm_http_api, the repository gets created in github, however, the project never gets created in my minishift environment. If I use a project named swarmhttpapi then no problems. Hope this is the right place to  report this.

**Comment by quintesse** [comment]
Correct. OpenShift is much more restrictive with its project names than GitHub. I've tightened the check to only allow OpenShift-compatible names. Closing.
